### PR TITLE
Contact table

### PR DESCRIPTION
**Pull request by andrewhojel**
working version of the contact table

* had to make some backend changes for it to work
* had to change the farm file to prepare for the table
* made basic updates to the table file

### CODE REVIEW

File: functions/APIs/farms.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ exports.getAllFarms = (request, response) => {
           farmId: doc.id,
           farmName: doc.data().farmName,
           location: doc.data().location,
-          hours: doc.data().hours,
+          locationId: doc.data().locationId,
+          // hours: doc.data().hours,
```
Comment (fatimazali) [inline functions/APIs/farms.js:19]: might not need this anymore

Comment (fatimazali) [reply functions/APIs/farms.js:19]: /maybe convert to a TODO format if its for future additions

File: view/src/components/produce.js [hunk 2]
```diff
@@ -303,7 +303,7 @@ class Produce extends Component {
               id="produce-name-search"
               options={data.map((produce) => produce.name)}
               value={value}
-              onSelect={this.handleSearch}  // Receive the name from data element for value
+              onSelect={this.handleSearch} // Receive the name from data element for value
```
Comment (fatimazali) [inline view/src/components/produce.js:306]: hahah oops @ this indent-spacing

File: view/src/components/farms.js [hunk 3]
```diff
@@ -755,11 +698,11 @@ class Farms extends Component {
...
                           </Typography>
                           <Typography variant="body2" component="p">
-                            Name: {farm.contactName}
+                            Name: {farm.contacts[0]["contactName"]}
                             <br />
-                            Phone: {farm.contactPhone}
+                            Phone: {farm.contacts[0]["contactPhone"]}
                             <br />
-                            Email: {farm.contactEmail}
+                            Email: {farm.contacts[0]["contactEmail"]}
```
Comment (jessimb) [inline view/src/components/farms.js:705]: are we guaranteed that contacts[0] will be defined? do we only want the first contact?

File: view/src/components/farms.js [hunk 4]
```diff
@@ -832,13 +775,13 @@ class Farms extends Component {
...
+                  {/* <Typography variant="body2" component="p">
+                    Name: {this.state.contacts[0]["contactName"]}
                     <br />
-                    Phone: {this.state.contactPhone}
+                    Phone: {this.state.contacts[0]["contactPhone"]}
                     <br />
-                    Email: {this.state.contactEmail}
-                  </Typography>
+                    Email: {this.state.contacts[0]["contactEmail"]}
+                  </Typography> */}
```
Comment (jessimb) [inline view/src/components/farms.js:784]: please remove commented out code

**Pull request merged**